priv/emqx.schema

```diff
@@ -2026,6 +2026,16 @@ end}.
   {datatype, flag}
 ]}.
 
+%% @doc performance toggle for subscribe/unsubscribe wildcard topic
```

Comment:
```suggestion
%% @doc performance toggle for subscribe/unsubscribe wildcard topic.
```

Reply:
thanks, This is a backport, will change it from upstream later.